tests/integration/projects/cpp_make_edge_cases_project/Makefile

```
@@ -0,0 +1,16 @@
+all: demo
+
+demo: src/main.o src/greeter.o
+	$(CXX) $(CXXFLAGS) -o demo src/main.o src/greeter.o
+
+src/main.o: src/main.cpp include/greeter.hpp
+	$(CXX) $(CXXFLAGS) -c src/main.cpp -o src/main.o
+
+src/greeter.o: src/greeter.cpp include/greeter.hpp
+	$(CXX) $(CXXFLAGS) -c src/greeter.cpp -o src/greeter.o
+
+clean:
```

checkmake (0.3.2)

[warning] Required target "all" must be declared PHONY.

(minphony)